sickbeard/logger.py

```diff
@@ -155,6 +155,19 @@ def submit_errors(self):
         if not (sickbeard.GIT_USERNAME and sickbeard.GIT_PASSWORD and len(classes.ErrorViewer.errors) > 0):
             self.log('Please set your GitHub username and password in the config, unable to submit issue ticket to GitHub!')
```

Comment:
Isn't there an issue to automatically turning `debug` on before sending an issue?